H/HiGHS/highs_common.jl

```diff
@@ -4,7 +4,7 @@ using BinaryBuilder, Pkg
 
 name = "HiGHS"
 
-version = v"1.6.0"
+version = v"1.6.1"
```

Comment:
The upstream version didn't change though?

Reply:
Yeah upstream hasn't released a new version yet. But I also thought that you couldn't add new dependencies to a `+1` version?

Reply:
Adding is actually fine, it's removing, or changing compat bounds, that doesn't work